### PR TITLE
Added Task Recurring to the Project

### PR DESCRIPTION
**Pull request by pepps01**

### CODE REVIEW

File: app/Http/Controllers/TaskController.php [hunk 1]
```diff
@@ -177,5 +177,41 @@ public function archived(Request $request)
...
+            }
+        }
+
+        return response()->json([
+            'message' => "Request Success",
+            'data' => $getArchive
+        ],200);
+    }
+    
+    public function createRecurringTask(Request $request, $id)
```
Comment (kayleb01) [inline app/Http/Controllers/TaskController.php:201]: The recurring task field is supposed to be a json which hold some data like when you want the task should keep recurring check the data structure [here](https://docs.google.com/document/d/1ncSNKNsVrd5fLekRpoy2TQYeB-J3g81bFw9poOtO6fU/edit)

File: app/Http/Controllers/TaskController.php [hunk 2]
```diff
@@ -145,18 +145,18 @@ public function taskcollection(){
 
     }
 
-    public function sort(Request $request)
```
Comment (kayleb01) [inline app/Http/Controllers/TaskController.php:148]: This isn't part of your work but i see that you have made changes to it, why?

File: app/Http/Controllers/TaskController.php [hunk 3]
```diff
@@ -177,5 +177,41 @@ public function archived(Request $request)
...
+        ],200);
+    }
+    
+    public function createRecurringTask(Request $request, $id)
+    {
+        $taskRecurring = $this->taskService->find($id);
+        $task = [];
+
+        if(isset($taskRecurring["recurring"])){
+            if($taskRecurring["recurring"][0] != "True"){
```
Comment (kayleb01) [inline app/Http/Controllers/TaskController.php:207]: I dont really get why you are selecting the first element of the recurring array when you set it to true only

**Comment by kayleb01** [comment]
Hi I've added comments in your code, please check and make modifications where neccessary, If you have questions please dm @Ericogia  on slack.

File: app/Http/Controllers/TaskController.php (hunk 2 above)
Comment (pepps01) [reply app/Http/Controllers/TaskController.php:148]: This was a shift space from what I can see. But I never touched the code.

File: app/Http/Controllers/TaskController.php (hunk 1 above)
Comment (pepps01) [reply app/Http/Controllers/TaskController.php:201]: But I don't see the data there.

**Pull request closed without merging**